packages/scaffolder/src/features/store.ts

```diff
@@ -251,6 +253,13 @@ export class FeatureStore {
   private static getLocalNpmPaths(): string[] {
     const paths: string[] = [];
 
+    // Retrieve the path to the node_modules using a known package.
+    const resolvedNodeModulesPath = require.resolve('chalk').match(/(.*node_modules)/)?.[0];
```

Comment:
Is this a known module relative to this package b/c `chalk` is installed as a dependency?